docs/tutorial/part-one/index.md

```diff
@@ -165,26 +167,27 @@ export default () => (
 )
 ```
 
-2.  Navigate to http://localhost:8000/about/.
+2. अब http://localhost:8000/about/ पर जाए।
 
 ![New about page](05-about-page.png)
 
-Just by putting a React component in the `src/pages/about.js` file, you now have a page accessible at `/about`.
+बस `src/Pages/about.js` फ़ाइल में एक React कौम्पोनॅन्ट डाल कर, अब आपके पास एक पेज `/about` उपलब्ध है।
 
-### ✋ Using sub-components
+### ✋ सब-कॉंपोनेंट्स का उपयोग करना
 
-Let’s say the homepage and the about page both got quite large and you were rewriting a lot of things. You can use sub-components to break the UI into reusable pieces. Both of your pages have `<h1>` headers — create a component that will describe a `Header`.
+मान लें कि होमपेज और about पेज दोनों काफी बड़े हो गए हैं और आप बहुत सारी चीजों को फिर से लिख रहे हैं। आप सब-कॉंपोनेंट्स का उपयोग करके UI को रियूज़बल पीसस मे ब्रेक कर सकते हैं। आपके दोनों पृष्ठों में `<h1>` हेडर्स है - एक कॉंपोनेंट बनाएं जो एक `Header`  का वर्णन करेगा।
```

Comment:
`कॉंपोनेंट` => `कौम्पोनॅन्ट`

change this everywhere

Reply:
changes done everywhere